src/main/java/com/liferay/faces/util/product/internal/ProductFactoryImpl.java

```diff
@@ -24,14 +24,14 @@
 
 
 /**
- * @author  Neil Griffin
  * @author  Kyle Stiemann
  */
 public class ProductFactoryImpl extends ProductFactory {
 
-	private static final Map<Product.Name, Product> PRODUCTS;
+	// Private Final Data Members
+	private final Map<Product.Name, Product> productMap;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.